test/test_vim.py

```diff
@@ -63,8 +64,9 @@ def test_strwidth():
 @with_setup(setup=cleanup)
 def test_chdir():
     pwd = vim.eval('getcwd()')
-    vim.chdir('/')
```

Comment:
Ahhh..yes...you can change the directory using '/' as the root path, which will then change the directory to the root drive, but then `:pwd` will print the drive, not `/`. I can edit the test to reflect this and make it more clear.